### PR TITLE
Add Survial example

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Example PR - do not merge

Mainly for @JamesKunstle - here's the promised workthrough of a basic Kaplan-Meier survival curve for the Augur repo. I've added some thoughts at the end on my interpretation of the curves as a community manager, feel free to shoot questions in my direction.

The Python lib you linked to earlier looks really nice, I look forward to seeing where you go with it ;)

### CODE REVIEW

File: notebooks/Augur Survival Example.ipynb [hunk 1]
```diff
@@ -0,0 +1,577 @@
+{
```
Comment (JamesKunstle) [inline notebooks/Augur Survival Example.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #8.      conf.int = TRUE,          # Add confidence interval
</pre><p>I'm so so glad that you included this, I loved the visualization of the confidence interval in your article. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sandiego-rh/sandiego/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (JamesKunstle) [inline notebooks/Augur Survival Example.ipynb:1]: <p>This is a phenomenal writeup- I'm going to use this as a reference document for sure. I really appreciate the time and care that you took with this, I think that it'll be excellent to apply this generally to any repo / groups of repo's we're interested in. </p><p> </p><p>One potential route is to learn whether issue survival or pr survival (or both) are correlated with project performance. i.e. take a very popular, successful project like the linux kernel- what does their pr/issue survival look like?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sandiego-rh/sandiego/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

**Comment by JamesKunstle** [comment]
@GregSutcliffe Would you be okay if we merged this draft? Cali and I agree that this is a great resource for R programmers and 8Knot ideas in the future.

**Comment by GregSutcliffe** [comment]
Sure thing. Happy to collaborate when you're ready to do something more "official" with it down the line ;)

**Pull request merged**